fingerprint-protection/fingerprint-protection-impl/src/main/java/com/duckduckgo/fingerprintprotection/impl/di/FingerprintProtectionModule.kt

```diff
@@ -50,7 +50,7 @@ object FingerprintProtectionModule {
     fun provideFingerprintProtectionDatabase(context: Context): FingerprintProtectionDatabase {
         return Room.databaseBuilder(context, FingerprintProtectionDatabase::class.java, "fingerprint_protection.db")
             .fallbackToDestructiveMigration()
-            .addMigrations(*ALL_MIGRATIONS)
+            .addMigrations(*ALL_MIGRATIONS.toTypedArray())
```

Comment:
I think you can remove the `.toTypedArray()` and make `ALL_MIGRATIONS` `arrayOf` instead of `listOf`